src/components/Books.js

```diff
@@ -1,44 +1,107 @@
 import React, {useState, useEffect} from 'react';
-import {getBookByBookId} from "../utils/getBook";
-import BookDetail from "./BookDetail";
-import BookCreate from "./BookCreate";
+import documentClient from '../configuredDocumentClient';
 
-const Books = (props) => {
+function useGetBook(bookId) {
```

Comment:
Just for my own personal understanding, is there a reason we should opt for defining functions with ```function``` instead of ```() =>```? Do you find this more legible, or is there another reason?

Reply:
Interesting you say this @BenMcH. I personally find the `function` style easier to read. Maybe it is because it looks more like other languages or because it stands out more from other variable declarations but I don't have a strong preference.